Matveibos/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+# this class is Hash
+class MyHash
+  include Enumerable
+
+  attr_reader :pair_count, :arr
```

Comment:
We can hide this methods in the private namespace, if we do not use them outside of this class